app/routes/wow.quantity-manipulation.tsx

```
@@ -0,0 +1,396 @@
+export const loader: LoaderFunction = async ({ request }) => {
+  const { getWoWSessionData } = await getUserSessionData(request)
+  const { server, region } = getWoWSessionData()
+
+  const params = new URL(request.url).searchParams
+
+  const validateFormData = z.object({
+    historicPrice: parseStringToNumber,
+    salesPerDay: z
+      .string()
+      .min(0.01)
+      .transform((value) => parseFloat(value)),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)